_test/test_utilities/test_range_add_border.m

```diff
@@ -0,0 +1,91 @@
+classdef test_range_add_border < TestCase
```

Comment:
why define `fe = eps` in most of these tests?

Reply:
because it was 4eps initially. May be reverted back if eps is insufficient